docker-compose.yml

```diff
@@ -1,16 +1,9 @@
 version: "2.3"
 services:
   androgee:
+    env_file: .env
     build:
       context: .
-      args:
-        egeeio_server:      $EGEEIO_SERVER
```

Comment:
The purpose of this `args:` block was to allow interpolations for each environment variable, so I don't have to hard code the variables (including the secrets) in a file. I do not believe the env files support interpolations like this

Reply:
Couldn't you just put the secrets into a `.env` file in the repo root.
Those are ignored in `.gitignore`.